src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Because they're variables so they have to be in different modules. It's the same with other window globals like `document`, `documentElement` as well as methods like `toString`, `indexOf` etc.